app/src/main/java/com/duckduckgo/app/browser/viewstate/OmnibarViewState.kt

```diff
@@ -20,6 +20,7 @@ data class OmnibarViewState(
     val omnibarText: String = "",
     val isEditing: Boolean = false,
     val shouldMoveCaretToEnd: Boolean = false,
+    val shouldMoveCaretToStart: Boolean = false,
```

Comment:
🔍 These properties feel odd to me as ViewState. I’d imagine a property for the caret position would be more like an index or enum. 

e.g.

```
    enum class CaretPosition {
        START,
        END,
    }
```

Things get iffy with EditTexts I realised, but these read more like events rather than state.

Reply:
Not a bad idea! I’m going to add them to the `OmnibarLayoutViewModel`, but leave the old `OmnibarViewState` as is.

Reply:
Actually no, this is not one or the other. Both states can be false, so we can’t use an enum for this.

Reply:
Would it make sense to have another option which is UNSET in that case?

```
    enum class CaretPosition {
        UNSET,
        START,
        END,
    }
```